### PR TITLE
.htaccess file

### PR DESCRIPTION
**Pull request by filisko**
Hi, I think that including the .htaccess file would be useful.

Regards,
Filis

### CODE REVIEW

File: public/.htaccess [hunk 1]
```diff
@@ -0,0 +1,5 @@
+RewriteEngine on
+RewriteCond %{REQUEST_FILENAME} !-d
+RewriteCond %{REQUEST_FILENAME} !-f
+
+RewriteRule . index.php [L]
```
Comment (shadowhand) [inline public/.htaccess:5]: Should this include `PT`?

Comment (shadowhand) [inline public/.htaccess:5]: https://httpd.apache.org/docs/current/rewrite/flags.html#flag_pt

Comment (filisko) [reply public/.htaccess:5]: I think that we should keep it simple enough to allow Equip to run properly, and if someone wants custom adjustments just leave it up to them.

**Comment by shadowhand** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/equip/project/pull-request/10/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=equip/project)

**Pull request merged**